torchgen/gen.py

```diff
@@ -2825,6 +2896,11 @@ def main() -> None:
         default=["headers", "sources", "declarations_yaml"],
         help="Generate only a subset of files",
     )
+    parser.add_argument(
+        "--only_backend",
```

Comment:
Is this option useful to produce code for another backend?

Reply:
removed